packages/ui/src/components/theme-selector-v2/index.tsx

```diff
@@ -0,0 +1,229 @@
+import { useEffect, useState } from 'react'
+
+import { Button, ButtonGroup, Dialog, Separator, Text } from '@/components'
+import { cn } from '@/utils/cn'
+
+export interface ThemeInterface {
+  mode: Mode
+  contrast: Contrast
+  colorAdjustment: ColorAdjustment
+  accentColor: AccentColor
+}
+
+export enum Mode {
+  Dark = 'Dark',
+  Light = 'Light'
+}
+
+export enum Contrast {
+  Default = 'Default',
+  HighContrast = 'High Contrast',
+  Dimmer = 'Dimmer'
+}
+
+export enum ColorAdjustment {
+  Default = 'Default',
+  Protanopia = 'Protanopia',
+  Deuteranopia = 'Deuteranopia',
+  Tritanopia = 'Tritanopia'
+}
+
+export enum AccentColor {
+  Yellow = 'yellow',
+  Red = 'red',
+  Pink = 'pink',
+  Purple = 'purple',
+  Blue = 'blue',
+  Cyan = 'cyan',
+  Orange = 'orange',
+  Brown = 'brown',
+  Green = 'green'
+}
+
+interface ThemeDialogProps {
+  defaultTheme?: ThemeInterface
+  theme?: ThemeInterface
+  open: boolean
+  onOpenChange: (open: boolean) => void
+  onChange: (language: Partial<ThemeInterface>) => void
+  onSave: (language: ThemeInterface) => void
+  onCancel: () => void
+  children: React.ReactNode
+}
+
+const ThemeDialog: React.FC<ThemeDialogProps> = ({
```

Comment:
This component doesn't match the naming of the file/directory.

Reply:
Done, file renamed.